### ISSUE TITLE
win10 - Data from the old version of the app is gone

### ISSUE THREAD

**Issue by TadaakiKutsuna** [Aftnet/OTPManager#11]
Hello.

I have updated OPT manager to a newer version.
When I started the app, I got the error message in the attached file.
I can't even seem to see the data I registered in the past.

Is there any way to recover this?

platform
windows 10 pro
Version 1909

![Err_MSG](https://user-images.githubusercontent.com/72126437/115831469-99d0cd80-a44c-11eb-9f96-11c23b3b5cee.png)

**Comment by mb1973** [comment]
same issue here. multiple installations affected

**Comment by shawty** [comment]
Yep exactly the same for me, and when I went to load my backup it refused, telling me that my password was corrupt, or the backup was corrupt or any one of half a dozen other variations of "something is corrupt"

on top of that... it now no longer sees my web-cam either, so I cannot put the code back in from the QRCode either.

Can't restore, Can't scan new codes, this app is now completely useless.....
